modules/deployment-scripts/aks-run-helm/README.md

```diff
@@ -0,0 +1,110 @@
+# AKS Run Helm Script
+
+An Azure CLI Deployment Script that allows you to run a Helm command at a Kubernetes cluster.
+
+## Description
+
+{{ Add detailed description for the module. }}
```

Comment:
Please could you add detailed description for the module?

Reply:
K, added some content and provided link for more details.